calendar_resource/models/calendar_event.py

```diff
@@ -64,38 +68,20 @@ def _format_datetime_interval_list(self, intervals):
                 'zallday': False,
                 'zduration': 24,
             }
-            datetimes += '\n%s\n' % self._get_display_time(**args)
-        return datetimes
+            datetimes.append(self._get_display_time(**args))
+        return '\n\n'.join(datetimes)
```

Comment:
hmm is it `.join` that doesn't work as expected, or is it using the double newlines that acts weird?